extensions/objects/protractor.js

```diff
@@ -205,6 +205,41 @@ var SanitizedUrlEditor = function(elem) {
   };
 };
 
+var SetOfNormalizedStringEditor = function(elem) {
+  const _getLength = async function() {
+    var items = (
+      await elem.all(by.repeater('item in localValue track by $index')));
```

Comment:
Can we use a CSS selector instead like we do in other files?

Also, need not use `await` for `elem.all` (see return [type](https://www.protractortest.org/#/api?view=ElementArrayFinder.prototype.all)).

Reply:
See comment above: https://github.com/oppia/oppia/pull/10882/files#r505887782